packages/babel-parser/src/parser/error-message.js

```diff
@@ -71,6 +71,8 @@ export const ErrorMessages = Object.freeze({
   InvalidLhs: "Invalid left-hand side in %0",
   InvalidLhsBinding: "Binding invalid left-hand side in %0",
   InvalidNumber: "Invalid number",
+  InvalidOrMissingExponent:
+    "Invalid or Missing exponent after 'e' in floating-point number",
```

Comment:
Thanks! Could you just make this lowercase? (don't use GH's "Apply changes" button because you need to regenerate the tests locally)

```suggestion
    "Invalid or missing exponent after 'e' in floating-point number",
```

Reply:
Or maybe just delete "Invalid or", I don't have a preference in either direction.

Reply:
Yeah I think we can drop "Invalid or"

Reply:
What do you guys think if I work around and make the error message more specific by figuring out if it is actually missing or invalid?

For example, `3ea` would also throw the same error message?
I would say it's invalid rather than saying missing exponent number....